app/views/events/show.html.erb

```diff
@@ -27,6 +27,12 @@
   <strong>Capacity:</strong>
   <%= @event.capacity %>
 </p>
+<p>
+  <strong>Demographics:</strong>
```

Comment:
still need to be hidden unless you're logged in.

Reply:
See comment from last closed pull request

Reply:
Kiera specifically said they need to be hidden unless you're logged in

Reply:
She didn't want people's personal information available to the public, so I think this is different.  I will contact her about it

Reply:
this is not different, she specifically said she didn't want the demographic information to be public. Don't contact her, that's unnecessary considering we had a meeting yesterday where she specifically said she didn't want demographic information to be public.